rps.js

```diff
@@ -0,0 +1,24 @@
+const playGame = function (userChoice) {
+    userChoice = userChoice.toUpperCase();
```

Comment:
there is no need to use toUpperCase()

in fact this is what is causing your code to let the player always lose.
because in your if functions it will be comparing ROCK and Scissors or PAPER and Paper.
in the computer ROCK and Rock are different variables.

be consistent with how you want to call your variables